Mage.Tests/src/test/java/org/mage/test/cards/single/rex/IndominusRexAlphaTests.java

```diff
@@ -0,0 +1,62 @@
+package org.mage.test.cards.single.rex;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ *
+ * @author jimga150
+ */
+public class IndominusRexAlphaTests extends CardTestPlayerBase {
+
+    @Test
+    public void testIndominusRexAlpha() {
+
+        addCard(Zone.HAND, playerA, "Indominus Rex, Alpha", 1);
+        addCard(Zone.HAND, playerA, "Ornithopter", 1);
+        addCard(Zone.HAND, playerA, "Rograkh, Son of Rohgahh", 1);
+        addCard(Zone.HAND, playerA, "Adorned Pouncer", 1);
+        addCard(Zone.HAND, playerA, "Ankle Biter", 1);
+        addCard(Zone.HAND, playerA, "Gladecover Scout", 1);
+        addCard(Zone.HAND, playerA, "Banehound", 1);
+        addCard(Zone.HAND, playerA, "Bontu the Glorified", 1);
+        addCard(Zone.HAND, playerA, "Aerial Responder", 1);
+        addCard(Zone.HAND, playerA, "Stonecoil Serpent", 1);
+        addCard(Zone.HAND, playerA, "Codespell Cleric", 1);
```

Comment:
For info: if not used in test yet then try to add card with multiple abilities (e.g. one card with multiple related abilities like Akroma, Vision of Ixidor must generate multiple counters).

Reply:
Already covered--ill add comments showing whats what though. multiple cards have more than one ability checked by indominus, and multiple pairs of cards overlap in abilities. expected behavior is that max of one counter per checked ability is added.